### DISCUSSION TITLE
Another CubeSatSim working de Larry W7DGP

### DISCUSSION THREAD

**Discussion by W7DGP** [alanbjohnston/CubeSatSim#185] [Show and tell]
Hooray, it works great! I set it up outside today in the sunshine and operated for 2 hours. It ran perfectly sending data indicating it was charging. Whenever I moved it into more sunshine, I got the Payload blue light indicating the accelerometer detecting movement. 
This was not my first project but parts of it were quite difficult: getting all the parts! What makes it possible to progress toward the finish are the videos and VERY complete build instructions. This is NOT a kit. I would say the easiest part was having all the software available for both computers. My biggest problems were not paying attention to the different sizes of headers and getting the board joined by headers so that it would fit the frame. I spent an entire day removing the Extra Tall Stacking Header from the Battery Board. When it came time to stack the boards, the headers would only go in so far and that left the stack too high for the frames. No problem though, I am using the top frame more as a cap rather than screwing it down. 
I have to say, seeing it working like a champ is very rewarding similar to homebrewing a transceiver. Check the images below. The telemetry is from 2 hours operation in sunlight. 4384 payloads decoded. The RT values shown were after I picked it up and brought it inside. Outdoors, the solar panels exposed to the sun (top or +Z and +X opposite the camera) would deliver about 150 ma each. It looked like it was running plus charging at about 200 mA. 
What's left?  Forgot about -Z, the bottom. I need to install a solar panel on the bottom. Also, I need more detailed info on the payload program and telemetry. After that, I want to come up with some payload experiments.
If anyone reading this would like info like measurements or lessons learned, don't hesitate to email me. I am glad to help.
73, Larry W7DGP 

![Screenshot 2022-08-28 125311_InputScreen](https://user-images.githubusercontent.com/109929525/187099747-ab6e446e-fa16-4ee4-a9d1-35aaeb2aab20.png)

![Screenshot 2022-08-28 After 3 hr in sun](https://user-images.githubusercontent.com/109929525/187099631-4409034d-84dd-42ed-9ad2-84566feca6cd.png)

**Comment by Boovtech** [comment]
This looks great!  Congratulations on a completed cubesat simulator!   I need to get mine out in the sun for a couple hours and see how the voltage on the batteries looks after that.

Thanks for sharing.

73
Virginia

**Comment by alanbjohnston** [comment]
Nice work!  Your solar panel battery charging is excellent.  Your FoxTelem looks perfect too!
